css-two-column-layout/styles.css

```diff
@@ -10,12 +10,38 @@ body {
   margin-right: auto;
 }
 
-.content {
+.display-row {
   display: flex;
+}
+
+.full {
+  text-align: center;
+  margin-left: auto;
+  margin-right: auto;
+  width: 60%;
+}
+
+.column-full {
+  display: flex;
+}
```

Comment:
`column` classes should only have `width` or `flex-basis` applied to them. If you think you need a column element with `display: flex` applied to it, what that likely really means is that you need a `row` inside of the `column` class element.